src/Whoops/Exception/Inspector.php

```diff
@@ -142,22 +143,7 @@ protected function getTrace(\Exception $e)
             return $traces;
         }
 
-        switch ($e->getSeverity()) {
-            case E_ERROR:
-            case E_RECOVERABLE_ERROR:
-            case E_PARSE:
-            case E_CORE_ERROR:
-            case E_COMPILE_ERROR:
-            case E_USER_ERROR:
-                $fatal = true;
-                break;
-
-            default:
-                $fatal = false;
-                break;
-        }
-
-        if (!$fatal) {
+        if (!Misc::isLevelFatal($e->getCode())) {
```

Comment:
Agreed, sorry I was thinking about a regular exception and not an error exception but we return those on line 143 already so we can change it back to getSeverity